samples/guestbook/wire_gen.go

```diff
@@ -6,28 +6,28 @@
 package main
 
 import (
-	"context"
-	"database/sql"
-	"github.com/aws/aws-sdk-go/aws/client"
-	"github.com/aws/aws-sdk-go/aws/session"
-	"github.com/go-sql-driver/mysql"
-	"github.com/google/go-cloud/blob"
-	"github.com/google/go-cloud/blob/fileblob"
-	"github.com/google/go-cloud/blob/gcsblob"
-	"github.com/google/go-cloud/blob/s3blob"
-	"github.com/google/go-cloud/gcp"
-	"github.com/google/go-cloud/mysql/cloudmysql"
-	"github.com/google/go-cloud/mysql/rdsmysql"
-	"github.com/google/go-cloud/requestlog"
-	"github.com/google/go-cloud/runtimevar"
-	"github.com/google/go-cloud/runtimevar/filevar"
-	"github.com/google/go-cloud/runtimevar/paramstore"
-	"github.com/google/go-cloud/runtimevar/runtimeconfigurator"
-	"github.com/google/go-cloud/server"
-	"github.com/google/go-cloud/server/sdserver"
-	"github.com/google/go-cloud/server/xrayserver"
-	"go.opencensus.io/trace"
-	"net/http"
+	context "context"
```

Comment:
This looks like you might be using an old version of wire, can you regenerate this file with the latest?

Reply:
Done.